.github/workflows/release.yml

```diff
@@ -389,6 +390,26 @@ jobs:
             workspace/runtime-*.yml
             workspace/emojivoto-demo.yml
             workspace/mysql-demo.yml
+      - name: AWS login (IAM role)
+        uses: aws-actions/configure-aws-credentials@f24d7193d98baebaeacc7e2227925dd47cc267f5 # v4.2.0
+        with:
+          role-to-assume: arn:aws:iam::795746500882:role/ContrastPublicBucketRW
+          aws-region: eu-central-1
+      - name: Upload pre-release artifacts to S3 bucket contrast-public
+        run: |
+          unix=$(date +%s)
+          files=(
+            workspace/contrast-cli/bin/contrast
+            workspace/contrast-enterprise-cli/bin/contrast-enterprise
+            workspace/coordinator*.yml
+            workspace/runtime-*.yml
+            workspace/emojivoto-demo.yml
+            workspace/mysql-demo.yml
+          )
```

Comment:
Could we download the artifact produced earlier as zip, and then loop over the contained files? I fear that it's easy to forget updating paths consistently.

Reply:
Sadly, we already have this block twice, so I think adding it a third time doesn't make things work. Would like to have this in a step as single source of truth and then consume it, but $time.